Galaxy.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<title>Galaxy</title>
+		<link rel="stylesheet" href="visual.css">
+		<link rel="shortcut icon" href="icon.ico" type="image/x-icon">
+	</head>
+	<body>
+	<header class="header">
+		
+		<nav class="bar">
+			<a href="#">Home</a>
+			<a href="#">About us</a>
+			<a href="#">Charity</a>
+			<a href="#">Contacts</a>
+			<p>UNKNOWN UNIVERSE</p>
+		</nav>
+		
+	</header>
+		
+		<div class="whole-universe">
+			<aside class="voating">
+					<section><input type="radio" name="planet" value="1"><i>Uranus</i></section>
+					<section><input type="radio" name="planet" value="2"><i>Mars</i></section>
+					<section><input type="radio" name="planet" value="4"><i>Phobos</i></section>
+					<section><input type="radio" name="planet" value="5"><i>GJ 273 b</i></section>
+					<section class="voting1"><input type="submit" value="Choose your destiny"></section>
+					<h3>«That's one small step for a man, one giant leap for mankind».(с)Neil Armstrong</h3>
+			</aside>	  
+			<article class="content">
```

Comment:
this should be a `section`

Reply:
> this should be a `section`

<article class="content"> change for <section class="content">